### ISSUE TITLE
Add more WALES and Pythons to the graphic

### ISSUE THREAD

**Issue by ntoll** [PyconUK/2016.pyconuk.org#42]
The "hero" graphic is awesome.

However, there's nothing that tells me it's in Wales. Surely the conference should reflect the locale of the location? It could be any random Victorian town hall in the background. Why not add...
- Something Welsh in a speech bubble from the knight (the Welsh for "hello, welcome to PyconUK" is "Mae fy hofrenfad yn llawn o lysywod")
- A DRAGON.
- A male-voice choir.
- Daffs
- Welsh cakes.

Also, there's nothing I can see that tells me this is a graphic for a Python conference.

**Comment by ghickman** [comment]
@ntoll – I took one look at that translation and thought… Nick's having us on isn't he?! Well played sir! I'd be in favour of keeping the translation as an easter egg for the intrepid [google] translator…

**Comment by drvinceknight** [comment]
> "Mae fy hofrenfad yn llawn o lysywod"

(Just in case: that's not the correct translation. Sat with a Welsh speaker here: @geraintpalmer who is also offering to write some Welsh if we wanted it - not the entire website, he doesn't have the time, but happy to contribute a sentence or two.)

**Comment by zeth** [comment]
"Also, there's nothing I can see that tells me this is a graphic for a Python conference."

Anyone got a photo of the debug duck?

**Comment by ntoll** [comment]
@zeth I believe @hawkz has one already.

**Comment by hawkz** [comment]
On it

**Comment by inglesp** [comment]
This was addressed by #116.

Unfortunately @hawkz, on some pages (including the homepage) the PyCon UK snakes are obscured by the title text.

Would it be possible/easy to reposition these?

**Comment by hawkz** [comment]
Yeah did spot that, but it looks ok at some resolutions.. my plan is to keep adding things so regardless of what's covered it'll still look great.

What's next on the list to add?

**Comment by graingert** [comment]
@hawkz a DRAGON...

**Comment by ghickman** [comment]
@hawkz – some more options for you (after the dragon ;):
- rugby ball
- welsh cakes
- bara brith cake (not sure if that'd work well?)

**Comment by ntoll** [comment]
- A BRASS BAND
- TOM JONES
- DRUIDS FROM ANGLESEY
- WELSH SLATE
- TOM JONES
- LLAP GOGH (viz: http://www.llapgoch.org.uk/ - honest, it's a real thing)
- (TOM JONES)

Basically, TOM JONES all the things...! ;-)

**Comment by graingert** [comment]
The background should probably be some form of [Welsh castle](https://www.google.co.uk/search?q=Welsh+castle&tbm=isch)

**Comment by ntoll** [comment]
Ivor the Engine?
http://www.televisiontunes.com/Ivor_the_Engine.html ;-)

**Comment by inglesp** [comment]
Can y'all stop clogging up my inbox?

**Comment by ntoll** [comment]
Actually, even better... a Male Voice Choir singing a Pythonic cover of Tom Jones's "It's not Unusual..." (just for @inglesp) ;-)

```
It's not unusual to write code with anyone
It's not unusual to run tests with anyone
But when I see you coding Ruby with anyone
It's not unusual to see me cry
Oh I wanna' Py...
```

Viz: https://www.youtube.com/watch?v=yca2BCwAnBQ

:-D

**Comment by inglesp** [comment]
@hawkz, do you have any plans here?  If not, please close.
